app/views/messages/index.html.erb

```diff
@@ -19,9 +19,11 @@
 <section class="message--create">
   <div class="grid">
     <div class="grid__item width-two-thirds shift-one-sixth">
-      <%= form_for @message, remote: true, :html => {:class => "sendbar form-width--searchbar"}, builder: GcfFormBuilder do |f| %>
-      <%= f.gcf_send_message_field :body, "Send", autofocus: true, placeholder: "Send a text message" %>
-      <input name="client_id" type="hidden" value="<%= @client.id %>">
+      <%= form_for @message, remote: true, :html => {class: 'sendbar form-width--searchbar' }, builder: GcfFormBuilder do |f| %>
```

Comment:
This breaks the field layout; on mobile, the field stretches off screen:
![screen shot 2017-06-30 at 2 30 03 pm](https://user-images.githubusercontent.com/546665/27754746-c8f84db4-5da0-11e7-9b3b-253363b3a1b1.png)

On desktop, the button is misaligned with the field:
![screen shot 2017-06-30 at 2 30 20 pm](https://user-images.githubusercontent.com/546665/27754749-cef7a642-5da0-11e7-8998-07839a40646e.png)

this is how it should look:
![screen shot 2017-06-30 at 2 32 18 pm](https://user-images.githubusercontent.com/546665/27754767-f40df3c8-5da0-11e7-8c21-0f57c7166543.png)